libmscore/hairpin.cpp

```diff
@@ -101,13 +101,14 @@ void HairpinSegment::layout()
       const int _trck = track();
       Dynamic* sd = nullptr;
       Dynamic* ed = nullptr;
+      Segment* start = hairpin()->startSegment();
+      Segment* end = hairpin()->endSegment();
```

Comment:
These two lines could probably be moved at least under the `if (autoplace() && !score()->isPalette())` condition, these variables don't seem to be used anything else.

Reply:
good point, fixed